src/main/java/seedu/address/model/person/fields/enums/Genders.java

```diff
@@ -9,10 +9,15 @@ public enum Genders {
     FEMALE("female"),
     DNS("dns");
 
-    private final String gender;
+    public final String gender;
```

Comment:
Think this can be private from your subsequent usage.

Reply:
Gotcha. Change added